### PR TITLE
Allow code climate

### PR DESCRIPTION
**Pull request by Ladas**
Allow code climate

### CODE REVIEW

**Comment by durandom** [comment]
@jrafanie can you take a look?

the :red_circle: ci is due to something else.

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  csslint:
+    enabled: false
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
+      - javascript
+  eslint:
+    enabled: true
```
Comment (jrafanie) [inline .codeclimate.yml]: do you have javascript in this repo?  If not, disable this engine.

File: .codeclimate.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- "**.html"
+- "**.json"
+- "**.png"
+- "**.svg"
+- "**.xml"
+- "**.yaml"
+- "**.yml"
+- "locale/"
+- "spec/"
+- "tools/"
```
Comment (jrafanie) [inline .codeclimate.yml]: I'd only exclude things you actually have in this repo.  locale?  tools?  svg?  xml?, etc.

**Comment by jrafanie** [comment]
I don't know why any of these things would fail CI,  restarted the build.

File: .codeclimate.yml (hunk 2 above)
Comment (Ladas) [inline .codeclimate.yml]: done (locale, tools and xml are present)

File: .codeclimate.yml (hunk 1 above)
Comment (Ladas) [inline .codeclimate.yml]: done

**Comment by Ladas** [comment]
@jrafanie @durandom I've cleaned up the code climate yml, it should be ok now

**Comment by jrafanie** [comment]
🎉 🙇

**Pull request merged**